web/src/SidebarTriggerButton.tsx

```diff
@@ -2,6 +2,13 @@ import React, { PureComponent } from "react"
 import { TriggerMode } from "./types"
 import "./SidebarTriggerButton.scss"
 
+export enum TriggerButtonTooltip {
```

Comment:
this seems like a weird thing to make an enum, since it's not really a type. probably better to be a plain object, which is what people usually use for messages

Reply:
oh, sure, i just don't know the idiom here! Is the idiomatic thing:
```
export const TooltipAlreadyQueued = "Cannot trigger an update if resource is already queued for update."
export const TooltipManualResourcePendingChanges = "This manual resource has pending file changes; click to trigger an update."
...
```
or
```
export const TriggerButtonTooltip = {
  AlreadyQueued: "Cannot trigger an update if resource is already queued for update.",
  ManualResourcePendingChanges: "This manual resource has pending file changes; click to trigger an update.",
  ...
}
```
or something else?

}

Reply:
either is fine